src/i3ipc.ml

```diff
@@ -249,6 +271,23 @@ module Reply = struct
 
   let result_of_command_outcome { success; error } =
     if success then Result.Ok () else Result.Error (error |? "")
+
+  type binding_modes = string list
+    [@@deriving of_yojson, show]
+
+  type config = {
+    config : string
+  } [@@deriving of_yojson, show]
```

Comment:
The i3 documentation seems to imply that there might be more fields in the future. So we should use `{ strict = false }` here.